clang/test/CodeGenHLSL/builtins/RWBuffer-constructor.hlsl

```diff
@@ -1,12 +1,12 @@
-// RUN: %clang_cc1 -triple spirv-vulkan-library -x hlsl -emit-llvm -disable-llvm-passes -o - %s | FileCheck %s --check-prefix=CHECK-SPIRV
+// RUN: %clang_cc1 -triple dxil-pc-shadermodel6.3-library -x hlsl -emit-llvm -disable-llvm-passes -o - %s | FileCheck %s --check-prefixes=DXCHECK
+
+// RUN: %clang_cc1 -triple spirv-vulkan-library -x hlsl -emit-llvm -disable-llvm-passes -o - %s | FileCheck %s --check-prefixes=SPVCHECK
```

Comment:
Note that these `--check-prefixes` don't include "CHECK", so none of the "CHECK:" lines here are executed at all. This may also imply that this test is broken before your change.

Reply:
Yes it broken now when i include CHECK in the prefixes, so i will keep it as it is maybe for other pr

Reply:
A run line was removed, I think in error, with #88918. I have restored it here: #106588